Python/chapter03/p01_three_in_one/my_stack.py

```diff
@@ -0,0 +1,183 @@
+"""Python version 3.7.0
+3.1 - Three in one
+Describe how you could use a single array to implement three stacks
+"""
+
+import copy
+import unittest
+
+from dataclasses import dataclass
+from typing import Generic, TypeVar
+from typing import List, Optional, Iterator
+
+T = TypeVar('T')
+
+@dataclass
+class StackNode(Generic[T]):
+    data: T
+    next: 'Optional[StackNode[T]]'
+
+
+class MyStack:
```

Comment:
```suggestion
class MyStack(Generic[T]):
```

otherwise mypy will complain that this class is not generic when attempting to use `T` in L30

Reply:
Line 30 uses T in `StackNode[T]` but no error messages from mypy 🤔 .  With or without `Generic[T]`.  After some investigation, I cannot seem to find why this is the case.  I'm using mypy version `0.910`.

Either way, going to commit your suggested change 👍